packages/sdk/lib/rnx-preset.js

```diff
@@ -0,0 +1,26 @@
+const addSdkCapabilities = (deps, devDeps) => {
```

Comment:
can we add a small block comment on what's the purpose of this config? With links to the rnx-kit docs for example, for those not familiar